src/expressions/StaticContext.ts

```diff
@@ -6,6 +6,7 @@ import ExecutionParameters from './ExecutionParameters';
 import FunctionDefinitionType from './functions/FunctionDefinitionType';
 import { FunctionProperties } from './functions/functionRegistry';
 import UpdatingFunctionDefinitionType from './xquery-update/UpdatingFunctionDefinitionType';
+import { errXQST0049 } from './xquery/XQueryErrors';
```

Comment:
This is already merged, I would have expected GitHub would have noticed this is not a change :(